code/modules/client/client_defines.dm

```diff
@@ -267,3 +267,6 @@
 	/// Holder variables for viewport scaling.
 	var/last_view_x_dim = 7
 	var/last_view_y_dim = 7
+
+	/// Whether the client is viewing OOC information in the inspect menu or not.
+	var/viewing_ooc_info = TRUE
```

Comment:
on the client? not the panel itself? what if I am looking at multiple OOC informations at once?

Reply:
Then they'll both show OOC info. There's very little point in adding a state per panel, cause you're pretty unlikely to inspect the same person every time, and that's a bunch of times you'll be finding yourself hiding OOC info.